templates/open-tech-will-save-us.html

```diff
@@ -0,0 +1,43 @@
+{% extends "index.html" %}
+{% block content %}
+{% set episodes_data = load_data(path="/content/open-tech-will-save-us/otwsu.toml") %}
+{% set lastest_episode_date = episodes_data.episodes[0].date | date(format="%Y%m%d") | int %}
+{% set today = now() | date(format="%Y%m%d") | int %}
+{% if lastest_episode_date >= today %}
+    {% set next_episode = episodes_data.episodes | first %}
+{% endif %}
+
+<main>
+    <article class="content">
+        <header>
+            <img src="/open-tech-will-save-us/otwsu{{ episodes_data.episodes[0].code }}.png"
```

Comment:
I think instead of this `code` field we could just check the [length](https://tera.netlify.app/docs/#length) of the array here, and for the loop we can list the episodes oldest to latest and iterate over its [reverse](https://tera.netlify.app/docs/#reverse) or `{% set code = length - loop.index %}` or something like that.